docs/explanations/how_to_interpret_output_of_fix_plan_api.md

```diff
@@ -0,0 +1,222 @@
+* **plan**: The necessary plan that would need to be executed to fix the CVE. We will detail each
+  possible plan step on a following section.
+* **warnings**: Warnings that can happen when generating the plan. For example, if the CVE doesn't
+  have a fix available for a given package, this will be reflected here. We will detail the possible
+  warning in a following section.
+
+
+## What operations can appear in the plan array ?
+
+There are four distinct operations that can appear on the plan array. They are:
+
+* **APT Upgrade**: This operation details which package would need to be upgraded to fix the CVE.
+  The JSON representation for this step is the following one:
+
+  ```json
+  {
+    "data": {
+      "binary_packages": ["binary_package"],
+      "source_packages": ["source_package"]
+    },
+    "operation": "apt-upgrade",
+    "order": 1
+  }
+  ``` 
+
+  This step provide all the binary and source packages on that data object.
+
+* **Attach**: This operation details that the user will need to attach to a Pro subscription.
+  The JSON representation for this step is the following one:
+
+  ```json
+  {
+    "data": {
+      "reason": "required-pro-service"
+    },
+    "operation": "attach",
+    "order": 1
+  }
+  ```
+
+  The reason field details why an attach is needed. This can be either because the user has
+  an expired subscription or the fix requires a specific Pro service to upgrade an affected
+  package.
+
+* **Enable**: This operation details that the user needs to enable a specific Pro service.
+  The JSON representation for this step is the following one:
+
+  ```json
+  {
+    "data": {
+      "service": "esm-infra"
+    },
+    "operation": "enable",
+    "order": 1
+  }
+  ```
+
+  The data object contains which service will need to be enabled.
+
+* **NoOp**: This indicates that no operation need to be performed to address the CVE.
+  The JSON representation for this step is the following one:
+
+  ```json
+  {
+    "data": {
+      "status": "system-not-affected"
+    },
+    "operation": "no-op",
+    "order": 1
+  }
+  ```
+  
+  The data object will state why no operation is needed. This can either be because the CVE doesn't
+  affect the system or because the CVE is already fixed in the machine.
+
+## What type of warnings can be generated ?
+
+There are two distinct warning that can happen when executing the plan API:
+
+* **Package cannot be installed**: This happens when the endpoint identifies that a given package
+  cannot be installed (i.e. the user lacks a required APT source in the machine). This can be seen
+  in the following JSON representation:
+
+  ```json
+  {
+    "data": {
+      "binary_package": "binary_package",
+      "binary_package_version": "3.5.12-1ubuntu7.16",
+      "source_package": "source_package"
+    },
+    "order": 1,
+    "warning_type": "package-cannot-be-installed"
+  }
+  ```
+
+ The data object details the packages that cannot be installed and the version of the package.
```

Comment:
```suggestion
 The `data` object details the packages that cannot be installed, and the package versions.
```

Reply:
This should be singular, because it will always contain just one package. I will fix the setence